### PR TITLE
fix the little tail

### PR DESCRIPTION
**Pull request by WuJunkai2004**
Added the judgment of whether to break a line when adding a little tail

### CODE REVIEW

File: app/src/main/java/com/huanchengfly/tieba/post/activities/ReplyActivity.java [hunk 1]
```diff
@@ -297,8 +297,10 @@ private String getReplyContent() {
         }
         builder.append(editText.getText());
         if (getAppPreferences().getLittleTail() != null) {
-            builder.append("\n")
-                    .append(getAppPreferences().getLittleTail());
+            if( replyInfoBean.isSubFloor() == false || replyInfoBean.getReplyUser() == null ){
```
Comment (n0099) [inline app/src/main/java/com/huanchengfly/tieba/post/activities/ReplyActivity.java:300]: https://github.com/Starry-OvO/aiotieba/issues/67#issuecomment-1376006123

**Comment by n0099** [comment]
#149

**Comment by HuanCheng65** [comment]
master 分支先不合入了（

**Comment by n0099** [comment]
您可以[修改base branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

**Comment by WuJunkai2004** [comment]
我早上关掉了有关该问题的issue。
看起来master似乎要被弃用了，希望4.0能尽快面世
________________________________
发件人: ***@***.*** ***@***.***> 代表 n0099 ***@***.***>
发送时间: 2023年3月11日 19:59
收件人: HuanCheng65/TiebaLite ***@***.***>
抄送: Wu Junkai ***@***.***>; Author ***@***.***>
主题: Re: [HuanCheng65/TiebaLite] fix the little tail (PR #147)


您可以修改base branch<https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request>

―

**Pull request closed without merging**